project-base/.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: bash
+dist: xenial
+stages:
+    - Linux
+    - MacOS
+services:
+    - docker
+os:
+    - linux
+jobs:
+    include:
+        -   stage: Linux
+            name: "Test linux install script"
+            script:
+                - echo 1 | ./scripts/install.sh
+                - docker-compose exec php-fpm ./phing checks tests-acceptance
+        -   stage: MacOS
+            name: "Test MacOS script in linux"
+            script:
+                - sudo apt install ruby
+                - gem install docker-sync
+                - sed -i -r "s#sed -i -E#sed -i -r#" ./scripts/install.sh
+                - mkdir -p ./var/elasticsearch-data
```

Comment:
This seems like something that should be part of `install.sh` right?

Reply:
Actually it is already there https://github.com/shopsys/shopsys/blob/2ba408e9d4df06c80ee74290776b995f5a962863/project-base/scripts/install.sh#L66

Anyway this is necessary because of user access rights to this folder on line below https://github.com/shopsys/shopsys/blob/2ba408e9d4df06c80ee74290776b995f5a962863/project-base/.travis.yml#L24

Reply:
So chmod is necessary, but mkdir can be removed, right?

Reply:
You cannot chmod a folder which does not exist

Reply:
Of course, you are rigth